src/loki/polygon_search.cc

```diff
@@ -41,16 +37,10 @@ ring_bg_t PBFToRing(const valhalla::Options::Ring& ring_pbf) {
   for (const auto& coord : ring_pbf.coords()) {
     new_ring.push_back({coord.lng(), coord.lat()});
   }
+  bg::correct(new_ring);
```

Comment:
whats that do? fix winding or self intersection?

Reply:
jep, our rings are expected as clockwise so it fixes counterclockwise rings. also closes open rings if the rings are typed that way (that we do ourselves actually while parsing the request, could remove that tiny bit now). 

I don't remember why, but I had `correct()` in an early version before merging the initial PR and I took it out. couldn't find any commit or so to figure out why I took it out. but since it compiles and tests pass, I'll leave it in (and remove the bit where we close the polygons in the request).